### PR TITLE
MAP map after srun

### PR DESCRIPTION
**Pull request by mand35**

### CODE REVIEW

**Review by pletzer — COMMENTED** [review]
Hi @mand35 

Does putting srun before map work for the MPI case as well? This lesson was written for the mpi case. If not then then I would recommend to write a note instead:

**Note** command "map" and its arguments must precede "srun" in the case of an MPI program. For serial or OpenMP programs we recommend "map" and its options to be after "srun".

**Comment by mand35** [comment]
This is a very good point. I just after my comment realized. 
Yes, for MPI application we need to add `map --profile` in front of srun, otherwise we have multiple instances of Map. And for OpenMP we need srun in front otherwise Map complains about the MPI implementation.

**Comment by pletzer** [comment]
@mand35 Do you want to fix your branch with the comment about reversing the order?

**Comment by mand35** [comment]
yes changed the PR

File: _lessons/python-scatter/01b-profiling-map.md [hunk 1]
```diff
@@ -38,11 +38,13 @@ cd mpi
 
 ## Using the "Express Launch"
 
-To use MAP we need to load the *forge* module in our batch script and add `map --profile` in front of the parallel run statements. For example:
+To use MAP we need to load the *forge* module in our batch script and add `map --profile` in front of the executable statements. For example:
 ```
 module load forge
-map --profile srun python scatter.py
+map --profile map python scatter.py
```
Comment (pletzer) [inline _lessons/python-scatter/01b-profiling-map.md]: There is a typo. "map --profile map python scatter.py" should be "map --profile srun python scatter.py". Thanks for fixing.

**Pull request merged**